pkgs/tools/networking/shadowsocks-libev/default.nix

```diff
@@ -25,6 +25,26 @@ stdenv.mkDerivation rec {
     "-DCMAKE_SKIP_BUILD_RPATH=ON"
   ];
 
+  postPatch = ''
+    # https://github.com/shadowsocks/shadowsocks-libev/issues/2901
+    substituteInPlace CMakeLists.txt \
+      --replace '# pkg-config' \
+                '# pkg-config
+                 include(GNUInstallDirs)'
+    substituteInPlace cmake/shadowsocks-libev.pc.cmake \
+      --replace @prefix@ @CMAKE_INSTALL_PREFIX@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_BINDIR@ @CMAKE_INSTALL_FULL_BINDIR@ \
+      --replace '$'{exec_prefix}/@CMAKE_INSTALL_FULL_LIBDIR@ @CMAKE_INSTALL_FULL_LIBDIR@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_INCLUDEDIR@ @CMAKE_INSTALL_FULL_INCLUDEDIR@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_DATAROOTDIR@ @CMAKE_INSTALL_FULL_DATAROOTDIR@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_MANDIR@ @CMAKE_INSTALL_FULL_MANDIR@
+
+    # https://github.com/dcreager/libcork/issues/173 but needs a different patch (yay vendoring)
+    substituteInPlace libcork/src/libcork.pc.in \
```

Comment:
Why would pc file of a vendored library be relevant?

Reply:
idk @alexshpilkin